parachain/pallets/ethereum-beacon-client/src/mock.rs

```diff
@@ -86,11 +86,40 @@ pub mod minimal {
 				version: [3, 0, 0, 1], // 0x03000001
 				epoch: 0,
 			},
+			deneb: Fork {
+				version: [4, 0, 0, 1], // 0x04000001
+				epoch: 4294967295,
+			},
+		};
+		pub const DenebChainForkVersions: ForkVersions = ForkVersions{
+			genesis: Fork {
+				version: [0, 0, 0, 1], // 0x00000001
+				epoch: 0,
+			},
+			altair: Fork {
+				version: [1, 0, 0, 1], // 0x01000001
+				epoch: 0,
+			},
+			bellatrix: Fork {
+				version: [2, 0, 0, 1], // 0x02000001
+				epoch: 0,
+			},
+			capella: Fork {
+				version: [3, 0, 0, 1], // 0x03000001
+				epoch: 0,
+			},
+			deneb: Fork {
+				version: [4, 0, 0, 1], // 0x04000001
+				epoch: 0,
+			},
 		};
 	}
 
 	impl ethereum_beacon_client::Config for Test {
 		type RuntimeEvent = RuntimeEvent;
+		#[cfg(feature = "beacon-spec-minimal-deneb")]
```

Comment:
I don't see these features defined in `Cargo.toml`. Is it obsolete code?

Either way, I would prefer not to add these features. Perhaps there is another solution.

Reply:
Yeah, obsolete and removed.
https://github.com/Snowfork/snowbridge/pull/981/commits/53e4fa68f9f3408e6f7c1147b28f8625566f687e

Reply:
Seems Decun will be activated on testnet soon(end of this Month?). I'll regenerate all the test fixtures and do some more tests then, though it does not block merging this PR.